extensions/PhabBugz/lib/Feed.pm

```diff
@@ -382,7 +382,7 @@ sub process_revision_change {
 
     # Pre setup before making changes
     my $old_user = set_phab_user();
-    my $bug = Bugzilla::Bug->new({ id => $revision->bug_id, cache => 1 });
+    my $bug = $revision->bug;
```

Comment:
@dklawren Is this a good / safe thing to fix here?

Reply:
Yeah pretty much the same thing.